content/docs/sidebar.json

```diff
@@ -5,6 +5,29 @@
     "source": "index.md",
     "icon": "house"
   },
+  {
+    "label": "CML",
```

Comment:
it looks a bit strange right between home and other DVC-related section after all, wdyt?

Reply:
I had the same initial concern.

Reply:
yep, i'm going to take @jorgeorpinel's suggestion and move the CML section to the bottom